src/app/phase-tester-response/issue/issue.component.ts

```diff
@@ -19,7 +19,6 @@ export class IssueComponent implements OnInit {
     ISSUE_COMPONENTS.TESTER_RESPONSE,
     ISSUE_COMPONENTS.SEVERITY_LABEL,
```

Comment:
Just a note - the SEVERITY_LABEL, TYPE_LABEL, TESTER_POST values are repeated twice in this array